webservices/resources/sched_b.py

```diff
@@ -77,7 +77,7 @@ def build_query(self, **kwargs):
 
 @doc(
     tags=['disbursements'],
-    description=docs.EFILING_TAG,
+    description=docs.EFILING_TAG
```

Comment:
@ccostino - I'm never sure when a trailing comma is needed and when it isn't - is it a good guideline that a trailing comma is better for data structures and unnecessary for function calls?

Reply:
@lbeaufort that's the convention I've been following, mostly based on [PEP-8's recommendations for trailing commas](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)!  By and large that is the recommendation, though it does make the case for trailing commas in a function call if it is split across multiple lines.  I lean more toward consistency in that case, though, hence this particular change.

Speaking of consistency, [PEP-8 has a note about that too right at the start](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds). :-)